### PR TITLE
add history feature

### PR DESCRIPTION
**Pull request by Sonusiety**

### CODE REVIEW

File: app/src/main/java/com/github/sdpsharelook/history/HistoryFragment.kt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        }
+        requireView().findViewById<TextView>(R.id.recapDescription).text = text
+
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_history, container, false)
```
Comment (rombirli) [inline app/src/main/java/com/github/sdpsharelook/history/HistoryFragment.kt:89]: create binding FragmentHistoryBinding

**Review by rombirli — COMMENTED** [review]
Great !

**Review by rombirli — APPROVED** [review]
great !

File: app/src/main/java/com/github/sdpsharelook/history/HistoryFragment.kt (hunk 1 above)
Comment (AdrianUlrich) [reply app/src/main/java/com/github/sdpsharelook/history/HistoryFragment.kt:89]: You don't have to, but if you don't, then use the layout sensitive inheritance of the Fragment class, so you don't have to override onCreateView

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -145,6 +145,8 @@
     <string name="flag_of_language">flag_of_language</string>
     <string name="edit">edit</string>
     <string name="delete">delete</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (AdrianUlrich) [inline app/src/main/res/values/strings.xml:149]: Todo, read what android studio auto-generates for you

**Review by AdrianUlrich — APPROVED** [review]
Nice, simple, approved.

**Review by AdrianUlrich — CHANGES_REQUESTED** [review]
Write some tests thanks

**Comment by AdrianUlrich** [comment]
Also maybe sometimes merge main into this

**Review by AdrianUlrich — APPROVED** [review]
lgtm

**Review by villemontc — APPROVED** [review]

**Pull request merged**